src/PaymentSuite/PaymillBundle/Resources/views/Paymill/scripts.html.twig

```diff
@@ -66,26 +104,29 @@
 
         if (error) {
 
-            $('#paymill_payment_error').html(error.apierror);
-            $("#paymill_view_submit").removeAttr("disabled");
+            oPaymill.oErrorContainer.innerHTML = error.apierror;
+            oPaymill.oSubmit.disabled = 'false';
+
 
         } else {
 
             /**
              * Removing Credit Cart data before sending it to back
              */
-            $('#paymill_view_credit_card_1').val('0000');
-            $('#paymill_view_credit_card_2').val('0000');
-            $('#paymill_view_credit_card_3').val('0000');
-            $('#paymill_view_credit_card_4').val('0000');
-            $('#paymill_view_credit_card_owner').val('000');
-            $('#paymill_view_credit_card_expiration_month').val('1');
-            $('#paymill_view_credit_card_expiration_year').val('2013');
-            $('#paymill_view_credit_card_security').val('000');
-
-            $('#paymill-payment-error').hide();
-            $('#paymill_view_api_token').val(result.token);
-            $("#paymill_form").get(0).submit();
+
+            document.getElementById('paymill_view_credit_card_1').value = '0000';
+            document.getElementById('paymill_view_credit_card_2').value = '0000';
```

Comment:
document.queryselector it's slower thant getElementByID. Also getElementById has better compatiblity with all the browsers and phones.